### PR TITLE
MINOR: remove some specifying types in tool command

### PR DESCRIPTION
**Pull request by wenbingshen**
As the title.I checked some of the command-line tools and removed some of the specified types.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by wenbingshen** [comment]
:) @dajac Can you help review this minor pr?Thanks.

**Review by chia7712 — COMMENTED** [review]
@wenbingshen thanks for your patch. As you are doing cleanup for `AclCommand`, could you remove all unnecessary type declaration from this class?

**Comment by wenbingshen** [comment]
> @wenbingshen thanks for your patch. As you are doing cleanup for `AclCommand`, could you remove all unnecessary type declaration from this class?

Thanks for your comments.I rechecked and performed the relevant unit tests.Please review again.

**Review by chia7712 — APPROVED** [review]
@wenbingshen thanks for your patch. LGTM

**Comment by chia7712** [comment]
@wenbingshen I believe those failed tests are unrelated to you patch. However, could you merge trunk to trigger QA again? We all love green QA :)

**Comment by wenbingshen** [comment]
> @wenbingshen I believe those failed tests are unrelated to you patch. However, could you merge trunk to trigger QA again? We all love green QA :)

Thank you for your review, I am very happy to do this. :)

File: core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala [hunk 1]
```diff
@@ -1189,8 +1189,7 @@ object ReassignPartitionsCommand extends Logging {
    */
   def alterPartitionReassignments(adminClient: Admin,
                                   reassignments: Map[TopicPartition, Seq[Int]]): Map[TopicPartition, Throwable] = {
-    val results: Map[TopicPartition, KafkaFuture[Void]] =
-      adminClient.alterPartitionReassignments(reassignments.map { case (part, replicas) =>
+    val results = adminClient.alterPartitionReassignments(reassignments.map { case (part, replicas) =>
```
Comment (dajac) [inline core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala:1191]: nit: The indentation of the two lines below could be corrected now.

File: core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala [hunk 2]
```diff
@@ -1215,8 +1214,7 @@ object ReassignPartitionsCommand extends Logging {
   def cancelPartitionReassignments(adminClient: Admin,
                                    reassignments: Set[TopicPartition])
   : Map[TopicPartition, Throwable] = {
-    val results: Map[TopicPartition, KafkaFuture[Void]] =
-      adminClient.alterPartitionReassignments(reassignments.map {
+    val results = adminClient.alterPartitionReassignments(reassignments.map {
```
Comment (dajac) [inline core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala:1216]: ditto

**Review by dajac — COMMENTED** [review]
Thanks for the patch. Left two minor stylistic comments.

**Comment by wenbingshen** [comment]
> Thanks for the patch. Left two minor stylistic comments.

Thank you for your comments, I have modified it, can you see see it again? Thank you. :)

File: core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala [hunk 3]
```diff
@@ -1215,10 +1214,9 @@ object ReassignPartitionsCommand extends Logging {
   def cancelPartitionReassignments(adminClient: Admin,
                                    reassignments: Set[TopicPartition])
   : Map[TopicPartition, Throwable] = {
-    val results: Map[TopicPartition, KafkaFuture[Void]] =
-      adminClient.alterPartitionReassignments(reassignments.map {
-          (_, (None: Option[NewPartitionReassignment]).asJava)
-        }.toMap.asJava).values().asScala
+    val results = adminClient.alterPartitionReassignments(reassignments.map {
+      (_, (None: Option[NewPartitionReassignment]).asJava)
```
Comment (dajac) [inline core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala]: While we are here, could we replace `(None: Option[NewPartitionReassignment]).asJava` by `Optional.empty[NewPartitionReassignment]()` or `Option.empty[NewPartitionReassignment].asJava`? Personally, I prefer using `Optional` directly here as we use `Optional.of` above.

Comment (wenbingshen) [reply core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala]: Good idea.I updated the code, please review it again, thank you.

**Review by dajac — APPROVED** [review]
LGTM, pending Jenkins. Thanks!

**Comment by wenbingshen** [comment]
> LGTM, pending Jenkins. Thanks!

Many thanks for your help!

**Pull request merged**